README.md

```diff
@@ -238,6 +238,26 @@ You'll be redirected automatically to an Okta login page. Once you successfully
 
 This module integrates with Spring Security's OAuth support, all you need is the mark your application with the standard `@EnableOAuth2Client` annotation. 
 
+## Proxy
+
+If you're running your application (with this okta-spring-boot dependency) from behind a network proxy, you could add JVM args to your application like:
+
+```bash
+-Dhttps.proxyHost=host
+-Dhttps.proxyPort=port
+-Dhttps.proxyUser="user"          # optional
+-Dhttps.proxyPassword="password". # optional 
+```
+
+or, you could set it programmatically like:
+
+```java
+System.setProperty("https.proxyHost", "https://example-proxy.com");
+System.setProperty("https.proxyPort", "443");
+```
+
+Note: The above samples apply only for `HTTPS` proxy type. Replace `https` with your proxy type as applicable. See [here](https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html) for the complete list of properties.
```

Comment:
nit: I don't think we need to mention non-https proxy configuration.  All communication with Okta is over https.

We do need to mention that this does NOT work with WebFlux

Reply:
Addressed.